src/modules/qt/common.cpp

```diff
@@ -87,8 +98,19 @@ void convert_qimage_to_mlt_rgba(QImage *qImg, uint8_t *mImg, int width, int heig
 #endif
 }
 
-void convert_mlt_to_qimage_rgba(uint8_t *mImg, QImage *qImg, int width, int height)
+void convert_mlt_to_qimage(
+    uint8_t *mImg, QImage *qImg, int width, int height, mlt_image_format format)
 {
+#if QT_VERSION >= QT_VERSION_CHECK(5, 12, 0)
+    // QImage::Format_RGBA64 was added in Qt5.12
+    if (format == mlt_image_rgba64) {
+        *qImg = QImage(mImg, width, height, QImage::Format_RGBA64);
+        return;
+    }
+#endif
+
+    Q_ASSERT(format == mlt_image_rgba);
+
 #if QT_VERSION >= QT_VERSION_CHECK(5, 2, 0)
```

Comment:
Another place to remove before Qt 5.2 code

Reply:
Done